Cargo.lock

```diff
@@ -1585,7 +1585,7 @@ checksum = "6e63cff320ae2c57904679ba7cb63280a3dc4613885beafb148ee7bf9aa9042d"
 
 [[package]]
 name = "squawk"
-version = "0.18.0"
+version = "0.19.0"
```

Comment:
Looks like this was missed in #269 and added automatically. Should I revert or keep?